matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/rendezvous/model/RendezvousIntent.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.rendezvous.model
+
+import com.squareup.moshi.Json
+
+enum class RendezvousIntent {
```

Comment:
Not sure but I think we should annotate it with `@JsonClass(generateAdapter = false)`? This is done for all enums which have `@Json` annotated fields. Maybe related to R8 issues @ganfra or @bmarty  could you confirm? Have you tried a release version in your tests? And if we need this, please ensure to cover all the other enums introduced by this PR.

Reply:
I've now added these. I don't recall if I did a release test yet or not.

Reply:
Removed.

Reply:
It should be `@JsonClass(generateAdapter = false)` not `@JsonClass(generateAdapter = true)`.